### PR TITLE
Shantanu_Assignment_19

### PR DESCRIPTION
**Pull request by ShantanuGTechnocredits**
image
image
image
image

### CODE REVIEW

**Comment by nehatawari** [comment]
Looks good

File: src/shantanu/assignment19/WordWithMaxLength.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package shantanu.assignment19;
+
+public class WordWithMaxLength {
+	
+	String splitString(String str) {
+		int maxLength = 0;
+		String maxString = " ";
```
Comment (harshvegada) [inline src/shantanu/assignment19/WordWithMaxLength.java:7]: blank should not be here

File: src/shantanu/assignment19/WordWithMinLength.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
+package shantanu.assignment19;
+
+public class WordWithMinLength {
+	
+	String splitString(String str) {
+		String minString = " ";
+		String[] arr = str.split(" ");
```
Comment (harshvegada) [inline src/shantanu/assignment19/WordWithMinLength.java:7]: same as above

**Review by harshvegada — APPROVED** [review]
**`Code logic looks good to me!`**

**Pull request merged**